src/CustomStyleCommand.ts

```diff
@@ -551,7 +552,7 @@ export const compareAttributes = (mark, style): boolean => {
     case MARKTEXTCOLOR:
       return mark.attrs['color'] === style[COLOR];
     case MARKFONTSIZE:
-      return mark.attrs['pt'] === Number(style[FONTSIZE]);
+      return Number(mark.attrs['pt']) === Number(style[FONTSIZE]);
```

Comment:
fuzzy compare works on numbers and a string of a number. So if you use fuzzy compare, you don't need to do the conversion to number.